modules/hello-world/pages/start-using-sdk.adoc

```diff
@@ -48,33 +49,38 @@ sudo pip3 install couchbase
 [source,bash]
 ----
 # Only needed during first-time setup:
-sudo yum install gcc gcc-c++ python-devel python-pip cmake
+sudo dnf install gcc gcc-c++ python3-devel python3-pip cmake
```

Comment:
If we're going to remove yum and apt-get from the SDK docs, we should remove it from all of the 3.0 docs simultaneously.
(also LCB, that is)
...we can do this for apt, but not for dnf, which does not ship with RHEL 7:
https://issues.couchbase.com/browse/DOC-7278

Reply:
fair enough.  yum still works.